src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/ApprovedLicenceSummary.kt

```diff
@@ -0,0 +1,59 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model
+
+import com.fasterxml.jackson.annotation.JsonFormat
+import io.swagger.v3.oas.annotations.media.Schema
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceKind
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceStatus
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceType
+import java.time.LocalDateTime
+
```

Comment:
This currently matches what we have as a licence in `ManagedCase` in `server/@types/managedCase.ts` in the frontend with the addition of the CRN. Not included any release date (conditional release date) in particular which I am still a bit on the fence on as the results are sorted by the conditional release date but we are not sending that back as part of the response?

Reply:
there's a risk that there were only enough things in that licence type that were required for compilation and that the nunjucks template referred to more. 

The type isn't very good - should really expand or refer to existing types

Reply:
As endpoint is not currently being used, will add changes to separate PR.